src/app/features/login/login.component.html

```diff
@@ -1 +1,18 @@
-<p>login works!</p>
+<div class="content-container">
+    <div class="header">
+      <div (click)="routeTo('/home');" class="left-header">
+          <img src="../../../../assets/svg/JBH_logo.svg" alt="J.B. Hunt Logo" class="logo">
+          <h3 class="brand-title">Hackathon</h3>
+      </div>
+      <div class="right-header">
+        <button (click)="routeTo('/home');" class="btn btn-small btn-yellow">Back</button>
+      </div>
+    </div>
+    <div class="forms-container">
+        <div class="options-container">
+            <button (click)="routeTo('/JGdrGCSnGnLyuXYfIRfEdUDA');" class="form-button">Judge</button>
```

Comment:
Is there where the Judge data is sent over to the back end?

Reply:
So it was not implemented in our version?

Reply:
Correct. I just left it the way they originally did it when they gave us the repo. But we could add in something similar to the code verification they do with the judges on their site that’s running.

Reply:
I think there is some implementation about that on the backend, so I'm confused now. I was going to suggest that you could follow the 'Judge' login process for the 'Admins'.

Reply:
I did think it was weird that there wasn’t any verification when I first tried it out. As for the suggestion, do you mean like a code to enter??

Reply:
I considered using their email and comparing it against a list of registered 'Judges' or 'Admins', but a code could work too. In the README, they talk about JWT, and I think that's what they are using to grant authorities. I think this is related to login and authentication.